src/pbt/__init__.py

```diff
@@ -370,9 +394,13 @@ def tag(path, repo_path, no_push, branch, custom):
     pbt.tag(repo_path, no_push=no_push, branch=branch, custom=custom)
 
 
-if __name__ == "pbt":
+def main():
     print(
         f"[bold purple]Prophecy-build-tool[/bold purple] [bold black]"
         f"v{pkg_resources.require('prophecy-build-tool')[0].version}[/bold black]\n"
     )
     cli()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Is this intentional? Earlier it was `if __name__ == "pbt":`. Not sure if this will have issue while running pbt from within prophecy.

Reply:
this is intentional, so we can test utils without invoking cli.__init__.

it should not affect, as long as we are invoking pbt.
I can release a test version if you want to test it from prophecy just to be sure.

Reply:
I have released a test version `prophecy-build-tool==1.2.32.dev0`
if you want to test it quickly.

Reply:
Given we only do one thing in this module, this change seems right